api/src/main/java/org/pmiops/workbench/api/BugReportController.java

```diff
@@ -0,0 +1,48 @@
+package org.pmiops.workbench.api;
+
+import javax.mail.Message;
+import javax.mail.Message.RecipientType;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.AddressException;
+import java.io.UnsupportedEncodingException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.Properties;
+import org.pmiops.workbench.firecloud.ApiException;
+import org.pmiops.workbench.model.BugReport;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BugReportController implements BugsApiDelegate {
+
+  private static final Logger log = Logger.getLogger(BugReportController.class.getName());
+
+
+
+  @Override
+  public ResponseEntity<BugReport> sendBug(BugReport bugReport) {
```

Comment:
To avoid too many terms, I might use either `send` or `sendBugReport` so we don't have to answer the question, "what's the difference between a bug and a bug report?"

Reply:
Sounds good. I'll move it to `sendBugReport` (I think more description rather than less would be good here.)